code/OAuthServerController.php

```diff
@@ -97,7 +128,7 @@ public function __construct()
         $grant->setRefreshTokenTTL(new DateInterval('P1M')); // refresh tokens will expire after 1 month
         $this->server->enableGrantType(
             $grant,
-            new DateInterval('PT1H') // access tokens will expire after 1 hour
+            new DateInterval($this->getGrantTypeExpiryInterval())
```

Comment:
`getGrantTypeExpiryInterval` is static method
```suggestion
            new DateInterval(self::getGrantTypeExpiryInterval())
```

Reply:
we're not using refreshtoken grant btw but yup can change this